application/__init__.py

```
@@ -26,15 +26,22 @@
-logger = logging.getLogger(__name__)
-logger.warning(f"Logging is set up with config={config}")
+except:
+    # Fallback to a basic configuration
+    logging.basicConfig(format='%(asctime)s - %(levelname)s - %(message)s')
+    logger.exception("Logging setup failed")
+else:
+    # Initialize the logger after dictConfig succeeds
+    # As well, to ensure that logging configuration succeeded regardless
+    # of a successful dictConfig execution, log a warning message
+    # to see whether the configuration was actually successful
+    logger.warning(f"Logging is set up with config={config}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)